raiden/blockchain/filters.py

```diff
@@ -142,7 +145,7 @@ def get_new_entries(self, target_block_number: BlockNumber) -> List[BlockchainEv
                 result.extend(
                     self._do_get_new_entries(from_block=from_block_number, to_block=to_block)
                 )
-                from_block_number += FILTER_MAX_BLOCK_RANGE
+                from_block_number += FILTER_MAX_BLOCK_RANGE  # type: ignore
```

Comment:
Instead of ignoring the typing, you can do
```suggestion
                from_block_number = BlockNumber(from_block_number + FILTER_MAX_BLOCK_RANGE)
```

A bit worse to read, but I would take that in preference of a type ignore.